test/fixture/long-stack-trace/package.json

```diff
@@ -0,0 +1,7 @@
+{
+	"ava": {
+		"name": "application-name",
+		"version": "0.0.1",
```

Comment:
You probably meant for these two fields to be at the top-level?

Node.js doesn't care about them though. Feel free to remove them.